src/scripts/main.js

```
@@ -1,3 +1,83 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const field = document.querySelector('.field');
+  const tbody = field.querySelector('tbody');
+
+  const appendRowBtn = document.querySelector('.append-row');
+  const removeRowBtn = document.querySelector('.remove-row');
+  const appendColBtn = document.querySelector('.append-column');
+  const removeColBtn = document.querySelector('.remove-column');
```

Button element lookups are used directly to attach listeners. You query the four buttons with `document.querySelector` (lines 7–10) then attach listeners; if any selector returns `null` the next `.addEventListener` will throw. The tests expect these elements to exist, so this is probably fine, but consider adding a small existence check (or a clear error) to fail fast and provide a helpful message if the markup is altered.